pkg/mock/server.go

```diff
@@ -20,3 +20,7 @@ type DynamicServer interface {
 	Stop() error
 	GetPort() string
 }
+
+const (
```

Comment:
Is there a constant class in the project to unify the definition and management of the constants used?

Reply:
I know what you mean. I put it here because I wish the `mock server` module could be independent. These constant variables will be referenced only during the `mock server` feature. Does that sound reasonable to you?

Reply:
got it, If it's only used in the current module, maybe he should be private?

Reply:
Sounds reasonable to me.